core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -225,6 +128,21 @@ <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-head
                 stroke-linecap="round"
                 stroke-width="12"
                 ng-attr-d="<[pathSvgParameters]>"/>
+          <image ng-if="!collectionPlaythrough.hasStartedCollection()" 
+                 x="175"
+                 y="100"
+                 height="100"
+                 width="100"
+                 ng-attr-xlink:href="<[getStaticImageUrl('/general/collection_start_here_arrow.svg')]>" 
+                 xlink:href=""></image>
+          <text ng-if="!collectionPlaythrough.hasStartedCollection()" 
+                x="200"
+                y="155"
+                alignment-baseline="middle"
+                font-family="Capriola, Roboto, Arial, sans-serif"
+                font-size="20"
+                fill="#E14738"
+                text-anchor="middle">Start Here!</text>
```

Comment:
Also, just to check -- do you want to fix this now or in a separate PR?

I made one more comment that should be addressed before merge, but I'm happy leaving this particular one to a separate PR if you like.